pom.xml

```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.9.0</version>
+      <version>2.9.5</version>
```

Comment:
Did 2.9.7 not work? I'd always recommend updating to the latest version if possible.